### DISCUSSION TITLE
Where does tea tree oil originate from?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#180] [Galaxy Trivia] [answered]
**Hint:** 🦘

**Comment by thuongtruong1009** [comment, marked as answer]
Tea tree oil, also known as melaleuca oil, originates from Australia. It is extracted from the leaves of the Melaleuca alternifolia tree, which is native to the northeast coast of New South Wales and southeast Queensland in Australia. The oil has been used for centuries by Australian Aboriginal people for its antiseptic properties and was later introduced to the wider world in the 1920s by Australian medical researcher Arthur Penfold
